packages/web-components/fast-foundation/src/breadcrumb-item/breadcrumb-item.spec.ts

```diff
@@ -0,0 +1,585 @@
+
+async function setup() {
+    const { element, connect, disconnect } = await fixture<FASTBreadcrumbItem>(
+        "fast-breadcrumb-item"
+    );
+
+    return { element, connect, disconnect };
+}
+
+describe("Breadcrumb item", () => {
+    it("should include a `role` of `listitem`", async () => {
+        const { element, connect, disconnect } = await setup();
+
+        await connect();
+
+        expect(element?.shadowRoot?.querySelector("[role='listitem']")).to.not.equal(
+            null
+        );
+
+        await disconnect();
+    });
+
+    it("should render `anchor` when `href` is provided", async () => {
+        const { element, connect, disconnect } = await setup();
+        const hrefExample: string = "https://fast.design";
+
+        element.href = hrefExample;
+
+        await connect();
+
+        expect(element.href).to.equal(hrefExample);
+        expect(element.shadowRoot?.querySelector("a")).not.to.equal(null);
+
+        await disconnect();
+    });
+
+    it("should NOT render `anchor` when `href` is not provided", async () => {
+        const { element, connect, disconnect } = await setup();
+
+        await connect();
+
+        expect(element.shadowRoot?.querySelector("a")).to.equal(null);
+
+        await disconnect();
+    });
+
+    it("should add an element with a class of `separator` when `separator` is true", async () => {
+        const { element, connect, disconnect } = await setup();
+
+        element.separator = true;
+
+        await connect();
+
+        expect(element.shadowRoot?.querySelector(".separator")).to.exist;
+
+        await disconnect();
+    });
+
+    it("should set the `href` attribute on the internal anchor equal to the value provided", async () => {
+        const { element, connect, disconnect } = await setup();
+        const hrefExample: string = "https://fast.design";
+
+        element.href = hrefExample;
+
+        await connect();
+
+        expect(element.shadowRoot?.querySelector("a")?.getAttribute("href")).to.equal(
+            hrefExample
+        );
+
+        await disconnect();
+    });
+
+    it("should set the `target` attribute on the internal anchor when `href` is passed", async () => {
+        const { element, connect, disconnect } = await setup();
+        const hrefExample: string = "https://fast.design";
+        const targetExample = "_blank";
+
+        element.href = hrefExample;
+        element.target = targetExample;
+
+        await connect();
+
+        expect(element.href).to.equal(hrefExample);
+        expect(element.shadowRoot?.querySelector("a")?.getAttribute("target")).to.equal(
+            targetExample
+        );
+
+        await disconnect();
+    });
+
+    it("should NOT set the `target` attribute on the internal anchor if `href` is not passed", async () => {
+        const { element, connect, disconnect } = await setup();
+        const targetExample = "_blank";
+
+        element.target = targetExample;
+
```

Comment:
I don't understand the need for this test, you already have a test for no internal anchor if href is not passed, in this case there shouldn't be an internal anchor (just like the previous test) let alone a target on that non-existent anchor. The reason I would suggest culling tests that don't prove anything new is around maintenance cost. perhaps this is a nit, but my suggestion is to take a look at the rest of these and see if some of them are testing stuff that you've already tested or you could tweak another test to accommodate all of one testable concept.

Reply:
I understand. I am keeping the test to set `target` on the anchor if href is set.